upflux_db.sql

```diff
@@ -0,0 +1,334 @@
+USE upflux;

```

Comment:
Can you add in the create database statement at the top but you can comment it out

Reply:
Thanks for the feedback. I have added that now.